app/models/search_builder.rb

```diff
@@ -75,9 +78,9 @@ def term_frequency_counts(solr_parameters)
     solr_parameters["sections.rows"] = 1_000_000
     sections_fl = ['id']
     transcripts_fl = ['id'] if transcripts_present
-   
-    # Add fields for each term in the query 
-    terms = solr_parameters[:q].split
+
+    # Add fields for each term in the query, explictly escape closing parenthesis to prevent error
+    terms = solr_parameters[:q].gsub(/(\))/, "\\\\\1").split
```

Comment:
Weird!  I wonder what makes closing parenthesis special that it needs this extra handling.  I tried out some additional cases and they all seemed to fail on closing parenthesis whether they were matched or unmatched.  Oddly open parenthesis don't seem to have a problem.
```
(example)
( Example )
Example ) captions
((Example))
(() Lisp is (great)())
```

Reply:
Just to clarify, is there additional work that needs to be done on the escaping here or were you just testing the original behavior?

And yeah, it is very weird that it is just specifically closing parenthesis/